packages/ui/package.json

```diff
@@ -11,7 +11,7 @@
     "build:all": "pnpm assets:web && pnpm assets:native && tsc --build ./tsconfig.json",
     "build:native": "pnpm assets:native && tsc -p ./tsconfig.native.json",
     "build:watch": "concurrently  \"pnpm panda:watch\"  \"pnpm build:native --watch --preserveWatchOutput\"  \"pnpm build:web --watch --preserveWatchOutput\"  \"pnpm build:all --watch --preserveWatchOutput\" ",
-    "build:web": "pnpm assets:web && tsc -p ./tsconfig.web.json",
+    "build:web": "pnpm assets:web && tsup --config tsup.config.web.ts",
```

Comment:
Do we still need the `assets` task if we're using tsup?

Reply:
We currently do but I will investigate configuring `tsup` to copy the assets over. 

I didn't setup `tsup` for the native components yet, only for web

Reply:
On initial investigation, we could add a `postbuild` script to copy the assets once `tsup` build finishes but its almost the same as just having the `assets` task. 

Right now `tsup` doesn't support moving the assets itself from what I can tell.

Reply:
I'll merge for now so I can continue testing, we can improve on later